app/services/course/assessment/submission/update_guided_assessment_service.rb

```diff
@@ -5,6 +5,8 @@ class Course::Assessment::Submission::UpdateGuidedAssessmentService <
   def update
     if params[:attempting_answer_id]
       submit_answer
+    elsif params[:attempting_question_id]
```

Comment:
Now that I think about it, attempting_question_id and attempting_answer_id can be used for other actions as well right? This might decrease the extensibility of the controller.

Maybe having separate fields with explicit checks for the action to do might allow us to use the same field for other actions in future?